index.html

```diff
@@ -802,8 +802,8 @@ <h3>DID URL Syntax</h3>
 
       <p>
 A <a>DID URL</a> is a network location identifier for a specific
-<a>resource</a>. It can be used to identify things like <a>DID subjects</a>,
-<a>verification methods</a>, <a>services</a>, specific parts of a <a>DID
+<a>resource</a>. It can be used to identify things like representations
```

Comment:
```suggestion
<a>resource</a>. It can be used to retrieve things like representations
```
I think the key point being made in this section is the ability to get to the resource, to download a digital representation of it in some way.